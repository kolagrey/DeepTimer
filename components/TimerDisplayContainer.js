import React from 'react';
import { View } from 'react-native';
import styles from '../styles';

const TimerDisplayContainer = (props) => {
    return (
        <View style={styles.displayContainer}>
            {props.children}
        </View>
    );
};

export default TimerDisplayContainer;