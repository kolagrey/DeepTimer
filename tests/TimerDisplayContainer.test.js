import React from 'react';
import { Text } from 'react-native';
import renderer from 'react-test-renderer';

import TimerDisplayContainer from '../components/TimerDisplayContainer';

describe('<TimerDisplayContainer/>', () => {

    test('renders correctly', () => {
        const tree = renderer.create(<TimerDisplayContainer/>).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it("is of type 'View'", () => {
        const tree = renderer
            .create(
                <TimerDisplayContainer />,
            ).toJSON();
        expect(tree.type).toBe('View');
    });

    it('can hold a child', () => {
        const tree = renderer
            .create(
                <TimerDisplayContainer>
                    <Text>
                        Hello Test
                    </Text>
                </TimerDisplayContainer>,
            ).toJSON();
        expect(tree.children.length).toBe(1);
    });
});