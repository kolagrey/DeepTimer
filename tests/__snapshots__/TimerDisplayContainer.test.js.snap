// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TimerDisplayContainer/> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "color": "#fff",
      "flex": 4,
      "justifyContent": "center",
    }
  }
/>
`;
